python/xorbits/_mars/services/task/supervisor/tests/test_task_manager.py

```diff
@@ -115,6 +118,7 @@ async def actor_pool():
             await MockStorageAPI.cleanup(pool.external_address)
             await MockClusterAPI.cleanup(pool.external_address)
             await MockMutableAPI.cleanup(session_id, pool.external_address)
+            shutil.rmtree(mars_tmp_dir, ignore_errors=True)
```

Comment:
Will `mars_tmp_dir` not be deleted during cleanup?

Reply:
The temp dir was removed in the end of `test_collect_task_info`, which is not reasonable. If the UT failed, the temp dir will not be removed.